soco/data_structures.py

```diff
@@ -690,6 +690,40 @@ def to_element(self, include_namespaces=False):
 
         return elt
 
+    def get_uri(self, resource_nr=0):
+        """The uri to use for playing this item.
+
+        Args:
+            resource_nr (int): The index of the resource. Note that there is no
+            known object with more than one resource, so you can probably keep
```

Comment:
When writing these docstring in the Napoleon style for Sphinx, a continued section should be indented once further. So it should be:
```python
    """
    Args:
        resource_nr (int): Very long multiline description of what it does
            it even continues on this line
            and this one as well.
    """
```